src/test/ui/nll/ty-outlives/impl-trait-captures.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0700]: hidden type for `impl Trait` captures lifetime that does not appear in bounds
+error[E0700]: hidden type for `Opaque(DefId(0:11 ~ impl_trait_captures[1afc]::foo::{opaque#0}), [ReStatic, T, ReEarlyBound(0, 'a)])` captures lifetime that does not appear in bounds
```

Comment:
side-note that this test uses `-Zverbose`, so not my fault and don't think it needs fixing.